### ISSUE TITLE
missing font on android?

### ISSUE THREAD

**Issue by nicolasburtey** [GaloyMoney/galoy-mobile#77]
**Describe the bug**
I've done a local build to test the Auth PR, and some font/images seems to be missing. I'm not sure yet if this is just on my laptop because it's a local build or if this is related to an update we may have done recently.

do you have the same issue? @cryptosalomao @jrbuhl93 

**Smartphone (please complete the following information):**
 - Device: Android Only. issue is not present on iOS

flag is missing on first screenshot. a lightning logo overlay is missing on the QRCode.

![IMG_5562](https://user-images.githubusercontent.com/11156370/124799792-8943d580-df1a-11eb-8f11-358ee423a2c8.jpg)

![IMG_5563](https://user-images.githubusercontent.com/11156370/124799774-847f2180-df1a-11eb-87ad-07a62b47194a.jpg)

**Comment by jrbuhl93** [comment]
This happens to me sometimes on first build on android. If I build a second time with react-native run-android it usually goes away.

**Comment by nicolasburtey** [comment]
weird ok. let me try again.

**Comment by nicolasburtey** [comment]
I also have the issue on iOS for the overlay on the QR Code
